packages/picasso/src/DatePicker/DatePicker.tsx

```diff
@@ -344,7 +345,11 @@ export const DatePicker = (props: Props) => {
 
   return (
     <>
-      <Container inline={width !== 'full'} ref={inputWrapperRef}>
+      <Container
+        inline={width !== 'full'}
+        ref={inputWrapperRef}
+        {...getDataPrivate(props)}
```

Comment:
I don't think you need this kind of spreading operation.
```suggestion
       data-private={props["data-private"]}
```
will do the same work.

Reply:
Do you need this `data-private` in all of these components: `Container`, `Input`, and `Calendar`?

Reply:
>I don't think you need this kind of spreading operation.

Will do. Somehow I missed react won't attach to the DOM those props that are explicitly set to `undefined`, so was trying to handle this smartly. Turned out I don't need it.

- [ ] pass data-private value explicitly instead of spreading

Reply:
> Do you need this `data-private` in all of these components: `Container`, `Input`, and `Calendar` at the same time?

Well, what this PR is about. when DatePicker is rendered with data-private I need both input and popup excluded from the video. And as popup works trough react portals (rendered outside of the subtree) I can't just put that onto a container. 

Though I might not need it on both popup container and calendar. Will check

- [ ] Get rid of extra data-privates if possible